src/bundle/Resources/config/default_settings.yaml

```diff
@@ -2,6 +2,10 @@ parameters:
     ezsettings.default.system_info_view: {}
     ezsettings.global.system_info_view:
         pjax_tab:
+            ez:
+                template: '@@ezdesign/system_info/platform.html.twig'
+                match:
+                    SystemInfo\Identifier: 'ez'
```

Comment:
As this is for master ee should find other names then ez and platform as of today :)